.github/workflows/ci.yml

```diff
@@ -55,11 +55,12 @@ jobs:
     if: ${{ needs.changes.outputs.v2 == 'true' || needs.changes.outputs.common == 'true' }}
     strategy:
       matrix:
+        rln_v2 : [true, false]
         os: [ubuntu-latest, macos-latest]
     runs-on: ${{ matrix.os }}
     timeout-minutes: 60
 
-    name: build-${{ matrix.os }}
+    name: build-${{ matrix.os }}-rln-v2-${{ matrix.rln_v2 }}
```

Comment:
> using the rln_v2 matrix this would be replaced by rln-v2-true or rln-v2-false?

correct: 
![image](https://github.com/waku-org/nwaku/assets/43716372/50c5a0b4-98f7-4789-8739-bf03924553cc)

Reply:
addressed in e140288b

Reply:
noice
![imagen](https://github.com/waku-org/nwaku/assets/8811422/feb7f236-19f8-4ad0-968c-f84b3b67b942)